GameServerSource/Assets/Packages/ExampleMessages/Mediators/NewPromoExampleMediator.cs

```diff
@@ -71,14 +68,10 @@ private void CheckForPromo()
                 }
                 NotifyClients(newsItem);
                 _titleNews.Add(newsItem);
-            }
-
+            }
+
+        }, null);
```

Comment:
Again, failures should go SOMEWHERE.
By default it probably still does nothing, but it should be possible to flip a boolean in settings, or a pound-define to enable debug logging. Don't care what, just don't blindly ignore errors.